src/thunderbots/software/ai/hl/stp/action/kick_action.cpp

```diff
@@ -59,7 +59,7 @@ void KickAction::calculateNextIntent(IntentCoroutine::push_type &yield)
     //                   |        \ /
     //                   >         A       <
     //                                     |
-    //                                     | DIST_TO_FRONT_OF_ROBOT / 2
+    //                                     | 0 dist (Point A is where the ball is)
```

Comment:
See above comment about this ASCII diagram

Reply:
Fixed ASCII diagram

Reply:
Fixed it again